src/main/scala/com/lunatech/iamin/database/Database.scala

```diff
@@ -0,0 +1,45 @@
+package com.lunatech.iamin.database
+
+import java.sql.Connection
+
+import cats.effect.{ContextShift, IO, Resource}
+import cats.implicits._
+import com.lunatech.iamin.config.DatabaseConfig
+import doobie.hikari.HikariTransactor
+import doobie.util.ExecutionContexts
+import liquibase.Liquibase
+import liquibase.database.jvm.JdbcConnection
+import liquibase.resource.ClassLoaderResourceAccessor
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success, Try}
+
+object Database {
+
+  def transactor(config: DatabaseConfig): Resource[IO, HikariTransactor[IO]] = {
+    implicit val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
I think it's better to pass `ContextShift` as an implicit parameter here. The point is: we need to control what thread pool to use externally.

Reply:
Also, not used anymore.